weave-js/src/components/Form/AutoComplete.tsx

```diff
@@ -145,15 +145,17 @@ type AdditionalProps = {
   size?: SelectSize;
   isDarkMode?: boolean;
   maxWidth?: number;
+  minWidth?: number;
   hasInputValue?: boolean;
+  inputRef?: React.MutableRefObject<HTMLDivElement | null>;
```

Comment:
**nitpick:** `inputRef` naming seems misleading if it's actually a div element under there?

Reply:
Maybe `inputDivRef`? or `autocompleteDivRef`?

Reply:
we should probably leave out "input" entirely. if you inspect the elements, there's an actual `input` element nested inside, but this ref is definitely not being attached to that.

have you tried `React.forwardRef()`?

Reply:
I did try the forwardRef, but I don't think it works well with generics 🤔 see https://github.com/wandb/core/pull/24247/files#r1777844929. We could try using the wrapper approach but it seems like the custom name is similar?

Reply:
ahh i see. maybe something like `containerRef` or `autocompleteRef` seems fine then!